### DISCUSSION TITLE
Install DiaNN package in R

### DISCUSSION THREAD

**Discussion by BenoitBe** [vdemichev/DiaNN#192] [Q&A]
Hello,

I have some trouble to install the package in R (4.1.1 and 3.6.1), it appear that the package is not available for these R versions. Which version do you use for it?

Many thanks in advance

**Comment by vdemichev** [comment]
Should work under R 3.6. Indeed, problems have been reported under R 4.0, apparently they broke backward compatibility with the update (yet again)...

**Comment by BenoitBe** [comment]
Thanks for your feedback vdemichev. I saw that instead you use iq. May you give me the R command line you use to process directly the DIANN report for MaxLFQ? (I am vey friendly wih R...)

I saw that you have to convert to log2?

**Reply by vdemichev** [reply]
Hi Benoit,

Sorry, I don't have a ready script for that, but as I recall that was pretty easy when I used iq.
In general, you only need to do that if you are not happy with MaxLFQ quantities produced by DIA-NN itself.

Best,
Vadim

**Comment by BenoitBe** [comment]
Thank again for your rapid answer Vadim and as I understand, DIA-NN already perform MaxLFQ :)

This is the one we have in the .pg_matrix.tsv?

I have an addtional question :) I have the script to do that, but the number of peptides per protein is not directly available? Actually I count them from the "total" report and add the column to the .pg_matrix.tsv before process into Perseus.

Thanks again

**Reply by vdemichev** [reply]
Yes, both pg_matrix and the main report. The easiest to count the number of peptides per protein is probably using pr_matrix.

Best,
Vadim
